shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/plants/presentation/detail/components/PlantDetailScreen.kt

```diff
@@ -226,7 +226,7 @@ private fun PlantDetails(
                     }
                 ) {
                     Text(
-                        text = UiText.StringRes(checkout_with_price, listOf(plant.price)).asString(),
+                        text = UiText.StringRes(checkout_with_price, listOf(plant.priceText)).asString(),
```

Comment:
I would suggest changing the button text to something like 'Add to Basket' for more clarity

Also, displaying the total sum of the selected quantity of items on the button rather than a single item price would be better in my opinion. Just an idea 😊

Reply:
Yes you are right